pyproject.toml

```diff
@@ -23,7 +23,6 @@ classifiers = [
     "Intended Audience :: Developers",
     "Natural Language :: English",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
```

Comment:
Should we include python 3.12?

Reply:
I was very confused about why it wasn't here... but it just happened to be right at the 'expand down' cut off 😄